### DISCUSSION TITLE
CVE-2023-5363 - OpenSSL 3.0.12 release

### DISCUSSION THREAD

**Discussion by sksr123** [openssl/openssl#23012] [Q&A] [answered]
Hello,

We need to upgrade OpenSSL version due to CVE-2023-5363. We are currently using 3.0.10 version.
Based on this advisory- https://www.openssl.org/news/secadv/20231024.txt

OpenSSL 3.1 and 3.0 are vulnerable to this issue.

OpenSSL 3.0 users should upgrade to OpenSSL 3.0.12.
OpenSSL 3.1 users should upgrade to OpenSSL 3.1.4.

Does the 3.0.12 download from 2023-Oct-24 include this fix?
openssl-3.0.12.tar.gz

Thanks.

**Comment by t8m** [comment, marked as answer]
Yes, CVE-2023-5363 is fixed in OpenSSL 3.0.12.

**Comment by sksr123** [comment]
Thank you
